### ISSUE TITLE
[Bug]: Toolbar - Radio - JAWS doesn't change to forms mode when user TAB into toolbar

### ISSUE THREAD

**Issue by kolaps33** [microsoft/fluentui#26422]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
System:
    OS: Windows 10 10.0.22621
    CPU: (12) x64 Intel(R) Xeon(R) E-2176M  CPU @ 2.70GHz
    Memory: 12.46 GB / 31.74 GB
  Browsers:
    Edge: Spartan (44.22621.1105.0), Chromium (109.0.1518.55)
    Internet Explorer: 11.0.22621.1
```

### Are you reporting Accessibility issue?

None

### Reproduction

https://codesandbox.io/s/bvvfkc?file=/example.tsx

### Bug Description

Repro Chrome+JAWS
run JAWS
1. set focus on button "to set focus"
2. press TAB key until focus is inside the toolbar

## Actual Behavior
JAWS doesn't change the into forms mode. Then user cannot use right/left arrows keys, because they are still in Virtual cursor mode. 

## Expected Behavior
as for others toolbars mode is switched into forms mode

Comment:
When I investigated it looks like it is related to tabster element. 
on this simple codesandbox:
[https://codesandbox.io/s/naughty-dream-z0pzvs](https://codesandbox.io/s/naughty-dream-z0pzvs)
tried change Tabster into input could help

### Requested priority

Low

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by kolaps33** [comment]
We hat talk about this bug already with @mshoho and with @jurokapsiar and considering it as low priority.

**Comment by kolaps33** [comment]
When I investigated it deeper it looks like JAWS expected behavior.
JAWS doesn't change the mode for radio group. 
[JAWS exiting auto forms mode and switching to Browse mode when tabbing to a radiogroup #574](https://github.com/FreedomScientific/standards-support/issues/574)
